File: 05/5.c
#include <stdio.h>

int isDivByRange20(long long n)
{
	long long i;
	for (i = 20 ; i >= 2 ; i--)
	{
		if(n % i != 0)
			return 0;
	}
	return 1;
}

int main()
{
	long long i = 20;
	while (! isDivByRange20(i))
		i += 20;
	printf("%d\n" , i);
}

File: 05/5.py
def main():
	i = 20
	while (not isDivByRange20(i)):
		i = i + 20
	print(i)

def isDivByRange20(n):
	for i in range(20 , 2 , -1):
		if (n % i != 0):
			return False
	return True;

main()
